src/main/java/com/wolox/training/model/Book.java

```diff
@@ -0,0 +1,144 @@
+package com.wolox.training.model;
+
+import static java.util.Objects.requireNonNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public final class Book {
+
+    private @Id
+    Long idBook;
+    private @Column
+    String genre;
+    private @Column(nullable = false)
```

Comment:
Let's use `@NotNull` instead of `@Column (nullable = false)`.
`@NotNull` checks that the property is not null before going to the database.